docs/changes/visualization/17444.bugfix.rst

```diff
@@ -0,0 +1 @@
+Fix broken `ticklabels` getter in `CoordinateHelper`. The getter was incorrectly returning the helper's ticks rather than the labels; this PR corrects that.
```

Comment:
The context of "this PR" will be lost once the changelog is assembled.
```suggestion
Fix ``CoordinateHelper.ticklabels``. The getter was incorrectly returning
the helper's ticks rather than the labels.
```